onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -505,7 +564,8 @@ public static ObaRegion getRegionFromBuildFlavor() {
         ObaRegionElement region = new ObaRegionElement(regionId,
                 BuildConfig.FIXED_REGION_NAME, true,
                 BuildConfig.FIXED_REGION_OBA_BASE_URL, BuildConfig.FIXED_REGION_SIRI_BASE_URL,
-                boundsArray, BuildConfig.FIXED_REGION_LANG, BuildConfig.FIXED_REGION_CONTACT_EMAIL,
+                boundsArray,new ObaRegionElement.Open311Server[0], BuildConfig.FIXED_REGION_LANG,
```

Comment:
For this to be set up properly from build variants, you'll need to add three new fields to `oba`, `agencyX`, and `agencyY` in `build.gradle`.

For `oba` and `agencyX`, make the values `null`:

`buildConfigField "String", "FIXED_REGION_OPEN311_JURISDICTION_ID", "null"`
`buildConfigField "String", "FIXED_REGION_OPEN311_API_KEY", "null"`
`buildConfigField "String", "FIXED_REGION_OPEN311_BASE_URL", "null"`

For `agencyY`, we'll hard code these values to the SeeClickFix test server (because the `agencyY` build variant is fixed to the Tampa OBA API):

`buildConfigField "String", "FIXED_REGION_OPEN311_JURISDICTION_ID", "null"`
`buildConfigField "String", "FIXED_REGION_OPEN311_API_KEY", "937033cad3054ec58a1a8156dcdd6ad8a416af2f"`
`buildConfigField "String", "FIXED_REGION_OPEN311_BASE_URL", "http://test.seeclickfix.com/open311/v2/"`

`ObaRegionElement.Open311Server[0]` should then be instantiated above, and you should pass in `jurisdictionId`, `apiKey`, `baseUrl` from `BuildConfig`, like:

~~~
ObaRegionElement.Open311Server[] open311Array = new ObaRegionElement.Open311Server[1];

ObaRegionElement.Open311Server open311Server;

if (FIXED_REGION_OPEN311_BASE_URL != null) {
        open311Server = new ObaRegionElement.Open311Server (
                BuildConfig.FIXED_REGION_OPEN311_JURISDICTION_ID, BuildConfig.FIXED_REGION_OPEN311_API_KEY,
                BuildConfig.FIXED_REGION_OPEN311_BASE_URL);
} else {
open311Server = new ObaRegionElement.Open311Server[0]
}
open311Array [0] = open311Server;

ObaRegionElement region = new ObaRegionElement(regionId,
                BuildConfig.FIXED_REGION_NAME, true,
                BuildConfig.FIXED_REGION_OBA_BASE_URL, BuildConfig.FIXED_REGION_SIRI_BASE_URL,
                boundsArray,
open311Array, 
BuildConfig.FIXED_REGION_LANG,
                BuildConfig.FIXED_REGION_CONTACT_EMAIL,
                BuildConfig.FIXED_REGION_SUPPORTS_OBA_DISCOVERY_APIS,
                BuildConfig.FIXED_REGION_SUPPORTS_OBA_REALTIME_APIS,
                BuildConfig.FIXED_REGION_SUPPORTS_SIRI_REALTIME_APIS,
                BuildConfig.FIXED_REGION_TWITTER_URL, false,
                BuildConfig.FIXED_REGION_STOP_INFO_URL);
~~~